cli/src/semgrep/constants.py

```diff
@@ -11,7 +11,7 @@
 DEFAULT_SEMGREP_CONFIG_NAME = "semgrep"
 DEFAULT_SEMGREP_APP_CONFIG_URL = "api/agent/deployments/scans/config"
 
-DEFAULT_TIMEOUT = 30  # seconds
+DEFAULT_TIMEOUT = 2  # seconds
```

Comment:
we might want to do that in a separate PR.
It's a pretty important change, deserving its own changelog.

Reply:
Agree, see #8879.